easybuild/easyblocks/n/netcdf.py

```diff
@@ -69,18 +69,22 @@ def configure_step(self):
 
         else:
             self.cfg.update('configopts', '-DCMAKE_BUILD_TYPE=RELEASE -DCMAKE_C_FLAGS_RELEASE="-DNDEBUG " ')
-            for (dep, libname) in [('cURL', 'curl'), ('HDF5', 'hdf5'), ('Szip', 'sz'), ('zlib', 'z')]:
+            for (dep, libname) in [('cURL', 'curl'), ('HDF5', 'hdf5'), ('Szip', 'sz'), ('zlib', 'z'),
+                                   ('PnetCDF', 'pnetcdf')]:
                 dep_root = get_software_root(dep)
                 dep_libdir = get_software_libdir(dep)
                 if dep_root:
                     incdir = os.path.join(dep_root, 'include')
                     self.cfg.update('configopts', '-D%s_INCLUDE_DIR=%s ' % (dep.upper(), incdir))
                     if dep == 'HDF5':
                         env.setvar('HDF5_ROOT', dep_root)
+                        self.cfg.update('configopts', '-DUSE_HDF5=ON')
                         libhdf5 = os.path.join(dep_root, dep_libdir, 'libhdf5.%s' % shlib_ext)
-                        self.cfg.update('configopts', '-DHDF5_LIB=%s ' % libhdf5)
+                        self.cfg.update('configopts', '-DHDF5_C_LIBRARY=%s ' % libhdf5)
```

Comment:
This is netCDF version dependent.
v4.3.0 uses HDF5_LIB, HDF5_HL_LIB, HDF5_INCLUDE_DIR
v4.4.0 is the first version to use HDF5_C_LIBRARY, HDF5_HL_LIBRARY, HDF5_INCLUDE_DIR

Please take this into account.

Reply:
Thanks for the warning, after reaching version 4.4 I was convinced that the names `HDF5_LIB` and  `HDF5_HL_LIB` were just an error in that comment. I updated the easyblock adding a dict with variable names per version to ease any additional changes that might be introduced in the future.